src/components/epic/EpicView.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {useState} from "react";
+import clsx from "clsx";
+import {LeftContainer} from "./leftContainer/LeftContainer";
+import {RightContainer} from "./rightContainer/RightContainer";
+import Card from "@material-ui/core/Card";
+import {useEpicStyles} from "./EpicStyles";
+import {createTaskList} from "../../api/EpicApi";
+
+export const EpicView = (props) => {
+    const classes = useEpicStyles();
+    const [taskLists, setTaskLists] = useState(props.epicData.taskLists);
+    const handlerAddTaskList = (epicId) => {
```

Comment:
Создание и хранение группы задач должно происходить в компоненте rightContainer, там где и располагается список задач
```suggestion
    const handlerAddTaskList = (epicId) => {
```

Reply:
Если создание и хранения группы задач происходит в RightContainer то при изменении state task group добавляется, но на клиенте компонент не реднерится заново. Пока оставил как было.

Reply:
Реализовал возможный вариант в ветке feature/epic-task-group. Ознакомься